drivers/adc/adc_lmp90xxx.c

```diff
@@ -807,6 +808,108 @@ int lmp90xxx_gpio_get_pin_value(struct device *dev, u8_t pin, bool *value)
 
 	return err;
 }
+
+int lmp90xxx_gpio_port_get_raw(struct device *dev, gpio_port_value_t *value)
+{
+	struct lmp90xxx_data *data = dev->driver_data;
+	u8_t tmp;
+	int err;
+
```

Comment:
We should also add
```
	if (k_is_in_isr()) {
		return -EWOULDBLOCK;
	}
```
to make sure the function returns immediately when called from ISR. Functions modifying the port value are also affected.

Reply:
I agree this driver is not isr-ok but IMO that should be fixed after this is merged.  Doing it correctly is somewhat complex (I believe sx1509b is finally right, but it took several passes).

We did not agree that isr-ok is a required property for GPIO operations, so it's out of scope.